src/esw/odrive/odrive.py

```diff
@@ -520,44 +442,30 @@ def __init__(self) -> None:
             'drive_vel_data', DriveVelData, queue_size=1)
 
     def drive_vel_cmd_callback(self, ros_msg: DriveVelCmd) -> None:
-        """Calls the change speeds function.
+        """
+        Calls the change speeds function.
 
         Note that this does NOT actually change speed that the ODrive comands
         the motors at. One must wait for the ODriveBridge._update() function
         to be called for that to happen.
-
-        Args:
-            ros_msg: A ROS message that has two floats that represents the
+        :param ros_msg: A ROS message that has two floats that represents the
             requested speeds for the left and right wheels.
-
-        Returns:
-            None.
         """
         if self._get_state_string() == "Armed":
             self._change_speeds(ros_msg.left, ros_msg.right)
 
     def ros_publish_data_loop(self) -> None:
-        """Continuously publishes velocity and current data.
-
-        Args:
-            None.
-
-        Returns:
-            None.
+        """
+        Continuously publishes velocity and current data.
         """
         while not rospy.is_shutdown():
             self._start_time = t.clock()
             self._publish_encoder_msg()
 
     def watchdog_while_loop(self) -> None:
-        """Continuously calls the update() function and checks if comms has
+        """
+        Continuously calls the update() function and checks if comms has

```

Comment:
Change to "Calls the update() function continuously" so that first word is always a descriptive-style verb